app/routes/auth.py

```
@@ -0,0 +1,247 @@
+        WeakPasswordError: If password doesn't meet requirements
+        InternalServerError: If registration fails
+        
+    Rate Limit:
+        5 requests per minute per client
+    """
+    try:
+        user = await create_user(profile_data, session)
+        return {
+            "message": "User created successfully",
+            "user": user
+        }
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)